src/test/java/acceptance/StationAcceptanceTest.java

```diff
@@ -0,0 +1,99 @@
+package acceptance;
+
+import static api.StationApiRequest.지하철역_리스트_조회;
+import static api.StationApiRequest.지하철역_삭제;
+import static api.StationApiRequest.지하철역_생성_요청;
+import static fixture.StationModifyRequestFixture.강남역;
+import static fixture.StationModifyRequestFixture.또다른지하철역이;
+import static fixture.StationModifyRequestFixture.새로운지하철역이름;
+import static fixture.StationModifyRequestFixture.지하철역이름;
+import static fixture.StationRequestFixture.지하철역_등록_요청_데이터_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import config.AcceptanceTestConfig;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import subway.service.response.StationResponse;
+
+@DisplayName("지하철역 관련 기능")
+class StationAcceptanceTest extends AcceptanceTestConfig {
+
+    /**
+     * When 지하철역을 생성하면 <br>
+     * Then 지하철역이 생성된다 <br>
+     * Then 지하철역 목록 조회 시 생성한 역을 찾을 수 있다
+     */
+    @DisplayName("지하철역을 생성한다.")
+    @Test
+    void createStation() {
+        // when
+        ExtractableResponse<Response> response = 지하철역_생성_요청(지하철역_등록_요청_데이터_생성(강남역));
```

Comment:
이것도 그냥 동일한 구문 같아요. 아래와 같이 요청 데이터를 생성하는 것을 `지하철역_생성_요청` 내부로 넣어도 될것 같아요 😅 

```suggestion
        ExtractableResponse<Response> response = 지하철역_생성_요청(강남역);
```